src/uwtools/resources/jsonschema/make-solo-mosaic.jsonschema

```diff
@@ -0,0 +1,46 @@
+{
+  "properties": {
+    "make_solo_mosaic": {
+      "additionalProperties": false,
+      "properties": {
+        "config": {
+          "additionalProperties": true,
+          "anyOf": [
+            {
+              "required": [
+                "dir"
+              ]
+            },
+            {
+              "required": [
+                "num_tiles"
+              ]
+            }
+          ],
```

Comment:
I think that says that at least one of these (but not necessarily both) is required. Is that right, or are they both required? If they're both required, we could replace this with a `"required": ["dir", "num_tiles"]` as a sibling to `"properties"`.

As mentioned above in an answer to you question, we could (and IMO should) add the other `config:` items here, specifying their types, but not adding them to `"required"`.